api-docs/paths/scim/member_details.json

```diff
@@ -0,0 +1,211 @@
+            {
+                "name": "organization_slug",
+                "in": "path",
+                "required": true,
+                "schema": {
+                    "type": "string"
+                }
+            },
+            {
+                "name": "member_id",
+                "in": "path",
+                "required": true,
+                "description": "The id of the member you'd like to update.",
+                "schema": {
+                    "type": "integer"
+                }
+            }
+        ],
+        "requestBody": {
+            "content": {
+                "application/json": {
+                    "schema": {
+                        "required": [
+                            "name"
+                        ],
+                        "type": "object",
+                        "properties": {
+                            "schemas": {
+                                "$ref": "../../components/schemas/scim/user.json#/definitions/schemas"
+                            },
+                            "emails": {
+                                "$ref": "../../components/schemas/scim/user.json#/definitions/emails"
+                            },
+                            "userName": {
+                                "$ref": "../../components/schemas/scim/user.json#/definitions/userName"
+                            },
+                            "active": {
+                                "$ref": "../../components/schemas/scim/user.json#/definitions/active"
+                            }
+                        }
+                    },
+                    "example": {
+                        "schemas": [
+                            "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+                        ],
+                        "Operations": [
+                            {
+                                "op": "replace",
+                                "value": {
+                                    "active": false
+                                }
+                            }
+                        ]
+                    }
+                }
+            },
+            "required": true
+        },
+        "responses": {
+            "204": {
+                "description": "Success"
+            },
+            "401": {
+                "description": "Unauthorized"
+            },
+            "403": {
+                "description": "Forbidden"
+            },
+            "404": {
+                "description": "Not Found."
+            }
+        },
+        "security": [
+            {
+                "auth_token": [
+                    "member:write"
+                ]
+            }
+        ]
+    },
+    "delete": {
+        "tags": [
+            "SCIM"
+        ],
+        "description": "Delete an organization member with a SCIM User DELETE Request.",
+        "operationId": "Delete an Organization Member",
+        "parameters": [
+            {
```

Comment:
seems like org_slug could be a reference that we could use everywhere, perhaps a future chore

Reply:
It could be another ref like `startIndex`.